src/index.ts

```diff
@@ -485,8 +496,10 @@ export async function assemble(config: Configuration, components: Components): P
 
   // Shorthands
   const shorthands: ShortHands = {
-    loadFileSystem: (username: string) => loadFileSystem({ config, username, dependencies: components }),
-
+    loadFileSystem: (username: string) =>
+      loadFileSystem({ config, username, dependencies: components }),
+    recoverFileSystem: (params: RecoverFileSystemParams) =>
+      recoverFileSystem({ agentDID: () => DID.agent(components.crypto), auth, dependencies: components, ...params }),
```

Comment:
i could maybe define `agentDID` before this section since it's used in two places now 🤔

Reply:
i ended up doing this